tests/tiup-dm/test_cmd.sh

```diff
@@ -15,6 +15,13 @@ tiup-dm --yes deploy $name $version $topo -i ~/.ssh/id_rsa
 
 tiup-dm list | grep "$name"
 
+# debug
```

Comment:
Will we delete this debug command lines finally?

Reply:
Unless we make clear why CI brokes

Reply:
OK, it's better to create an issue and link the issue here to help us track it and delete the debug command lines when the issue has been resolved.